build.gradle.kts

```diff
@@ -13,296 +16,50 @@
-                            }
-                        }
-                        developers {
-                            developer {
-                                id.set("smithy")
-                                name.set("Smithy")
-                                organization.set("Amazon Web Services")
-                                organizationUrl.set("https://aws.amazon.com")
-                                roles.add("developer")
-                            }
-                        }
-                        scm {
-                            url.set("https://github.com/smithy-lang/smithy.git")
-                        }
-                    }
-                }
-            }
-        }
-
-        // Don't sign the artifacts if we didn't get a key and password to use.
-        if (project.hasProperty("signingKey") && project.hasProperty("signingPassword")) {
-            signing {
-                useInMemoryPgpKeys(
-                    project.property("signingKey").toString(),
-                    project.property("signingPassword").toString()
-                )
-                sign(publishing.publications["mavenJava"])
-            }
-        }
-    }
-
-    tasks.register<Copy>("copyMavenMetadataForDevelopment") {
-        from("build/tmp/publishMavenJavaPublicationToMavenLocal") {
-            rename("module-maven-metadata.xml", "maven-metadata.xml")
-        }
-        val wdir = "${System.getProperty("user.home")}/.m2/repository/software/amazon/smithy/${project.name}"
-        into(wdir)
-    }
-
-    tasks.publishToMavenLocal {
-        finalizedBy("copyMavenMetadataForDevelopment")
-    }
-
-    // ==== CheckStyle ====
-    // https://docs.gradle.org/current/userguide/checkstyle_plugin.html
-    apply(plugin = "checkstyle")
-    tasks.named("checkstyleTest") {
-        enabled = false
-    }
-
-    // ==== Code coverage ====
-    // https://docs.gradle.org/current/userguide/jacoco_plugin.html
-    apply(plugin = "jacoco")
-
-    // report is always generated after tests run
-    tasks.test {
-        finalizedBy(tasks.jacocoTestReport)
-    }
-
-    // tests are required to run before generating the report
-    tasks.jacocoTestReport {
-        dependsOn(tasks.test)
-        reports {
-            xml.required.set(false)
-            csv.required.set(false)
-            html.outputLocation.set(file("$buildDir/reports/jacoco"))
-        }
-    }
-
-    // ==== Spotbugs ====
-    // https://plugins.gradle.org/plugin/com.github.spotbugs
-    apply(plugin = "com.github.spotbugs")
-    // We don't need to lint tests.
-    tasks.named("spotbugsTest") {
-        enabled = false
-    }
-    // Configure the bug filter for spotbugs.
-    spotbugs {
-        effort.set(Effort.MAX)
-        excludeFilter.set(file("${project.rootDir}/config/spotbugs/filter.xml"))
-    }
-}
-
-// The root project doesn't produce a JAR.
-tasks.named("jar") {
-    enabled = false
-}
-
-// ==== Javadoc ====
-afterEvaluate {
```

Comment:
I dont see any aggregate javadoc task anymore? Maybe I missed it, but we still need an aggregate javadoc task.